### PR TITLE
Move Plaid config into organization

### PR DESCRIPTION
**Pull request by dcblair**
## Description

> Give your teammates a high level overview of what's happening. What changes are being made? What is in scope or out of scope?

> Plaid config was outside of organization, in its own category. Moved it back into organization.

## Testing

> Teach your teammates how to run and check that your work does what you say it does.

- [ ] Is QA required?

## Related issues

> Help your teammates keep track of work. Link relevant PRs and tickets.
>
> You can link Linear tickets by ID within the title or add the entire Linear ticket URL inside the description. GH integrations will link tickets and PRs.

### CODE REVIEW

File: openapi.json [hunk 1]
```diff
@@ -2099,22 +2099,6 @@
         }
       }
     },
-    "/organization/address": {
```
Comment (dangdennis) [inline openapi.json:2102]: Looks like you moved address.

also make sure you run yarn build cuz there’s no changes so far in consolidated.json

**Review by dangdennis — CHANGES_REQUESTED** [review]
Looks incorrect

File: openapi.json (hunk 1 above)
Comment (dcblair) [reply openapi.json:2102]: Ah, I did it, then moved the config, again.
Thanks!

**Comment by dcblair** [comment]
> Looks incorrect

Can you specify?

File: openapi.json (hunk 1 above)
Comment (dangdennis) [reply openapi.json:2102]: How does plaid config bit now nest under Organization?

**Comment by dangdennis** [comment]
> > Looks incorrect
> 
> Can you specify?

Was referring to the yarn build step.

**Comment by dcblair** [comment]
> > > Looks incorrect
> > 
> > 
> > Can you specify?
> 
> Was referring to the yarn build step.

Ah, got it.

File: openapi.json (hunk 1 above)
Comment (dcblair) [reply openapi.json:2102]: It's above address, within org.

Comment (dcblair) [reply openapi.json:2102]: Screen Shot 2021-09-10 at 10 12 32 AM

Comment (dcblair) [reply openapi.json:2102]:
```  "/organization/credit_reporting": {
      "put": {
        "tags": ["credit_reporting"],
        "summary": "Configure credit reporting details for your organization",
        "description": "",
        "operationId": "configureCreditReporting",
        "requestBody": {
          "$ref": "#/components/requestBodies/ConfigureCreditReporting"
        },
        "responses": {
          "200": {
            "$ref": "#/components/responses/ConfigureCreditReporting"
          },
          "default": {
            "description": "Unexpected Error."
          },
          "401": {
            "description": "Unauthorized."
          },
          "403": {
            "description": "Forbidden."
          },
          "422": {
            "description": "Invalid input"
          },
          "429": {
            "description": "Too many requests."
          }
        }
      }
    },
    "/organization/plaid_config": {
      "put": {
        "tags": ["plaid_config"],
        "summary": "Update the Plaid client credentials for an organization",
        "operationId": "updateOrganizationPlaidConfig",
        "description": "",
        "requestBody": {
          "$ref": "#/components/requestBodies/UpdateOrgPlaidConfigInput"
        },
        "responses": {
          "200": {
            "description": "Updated Plaid credentials."
          },
          "default": {
            "description": "Unexpected Error."
          },
          "401": {
            "description": "Unauthorized."
          },
          "403": {
            "description": "Forbidden."
          },
          "404": {
            "description": "Not found."
          },
          "429": {
            "description": "Too many requests."
          }
        }
      }
    },
    "/organization/address": {
      "put": {
        "tags": ["address"],
        "summary": "Update the organization's address",
        "description": "",
        "operationId": "updateOrganizationAddress",
        "requestBody": {
          "$ref": "#/components/requestBodies/UpdateOrgAddress"
        },
        "responses": {
          "200": {
            "$ref": "#/components/responses/UpdateOrgAddress"
          }
        }
      }
    }
  },```

Comment (dcblair) [reply openapi.json:2102]: That didn't format perfectly.

**Review by dangdennis — APPROVED** [review]

**Pull request merged**